src/img/components/icons/pages-code.svg

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-   viewBox="0 0 32 32" style="enable-background:new 0 0 32 32;" xml:space="preserve">
-<path fill-opacity="0.2" d="M28,25h-4V6c0-0.6-0.4-1-1-1H10V3h18V25z"/>
-<path d="M6,13h14v1H6V13z M6,10h14v1H6V10z M18,22v-1.5c0-0.3-0.2-0.5-0.5-0.5H17v-1h1c0.6,0,1,0.4,1,1v2h1v1h-1v2c0,0.6-0.4,1-1,1
-  h-1v-1h0.5c0.3,0,0.5-0.2,0.5-0.5V23h0.2c0.3,0,0.5-0.2,0.5-0.5c0-0.3-0.2-0.5-0.5-0.5H18z M14,22v-1.5c0-0.3,0.2-0.5,0.5-0.5H15v-1
-  h-1c-0.6,0-1,0.4-1,1v2h-1v1h1v2c0,0.6,0.4,1,1,1h1v-1h-0.5c-0.3,0-0.5-0.2-0.5-0.5V23h-0.2c-0.3,0-0.5-0.2-0.5-0.5
-  c0-0.3,0.2-0.5,0.5-0.5H14z M29,1H9C8.4,1,8,1.4,8,2v3H3C2.4,5,2,5.4,2,6v24c0,0.6,0.4,1,1,1h20c0.6,0,1-0.4,1-1v-3h5
-  c0.6,0,1-0.4,1-1V2C30,1.5,29.6,1,29,1z M22,29H4V7h18V29z M28,25h-4V6c0-0.6-0.4-1-1-1H10V3h18V25z"/>
-</svg>
```

Comment:
I did intend to delete this file here, but can move to a separate PR if you'd prefer!